microcosm_logging/factories.py

```diff
@@ -57,6 +59,25 @@ def configure_logger(graph):
     return getLogger(graph.metadata.name)
 
 
+def enable_loggly(graph):
+    if graph.metadata.debug or graph.metadata.testing:
+        return False
+
+    if not graph.config.logging.loggly.enabled:
+        return False
+
+    try:
```

Comment:
I'm not sure we should just fall back to normal logging if loggly is enabled but not configured properly? Shouldn't we fail telling the user their stuff isn't set up right?

I'm guess we need this because loggly is enabled by default above?